### PR TITLE
update postgres version to 11 for local setup

### PR DESCRIPTION
**Pull request by matt-k1998**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-4290

### Change description ###
update postgres version to 11 for local setup

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: docker-compose-dependencies.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
       - microserviceKeys_ccd_case_document_am_api=AAAAAAAAAAAAAAAA
       - microserviceKeys_am_role_assignment_service=AAAAAAAAAAAAAAAA
       - TESTING_SUPPORT_ENABLED=true
-      - JAVA_TOOL_OPTIONS=-Xms128m -Xmx512m -XX:MaxPermSize=512m
+#      - JAVA_TOOL_OPTIONS=-Xms128m -Xmx512m -XX:MaxPermSize=512m
```
Comment (mdayican) [inline docker-compose-dependencies.yml:19]: why did comment out?

Comment (matt-k1998) [reply docker-compose-dependencies.yml:19]: It was causing an error in the "service-auth-provider-api" docker container causing it to quit. Something along the lines of JAVA_TOOL_OPTIONS option not recognised.

**Review by mdayican — APPROVED** [review]

File: docker-compose-dependencies.yml (hunk 1 above)
Comment (mdayican) [reply docker-compose-dependencies.yml:19]: is it related to postgres update ?

Comment (matt-k1998) [reply docker-compose-dependencies.yml:19]: no, it was happening beforehand anyway. Maybe this change should not be a part of this PR though?

Comment (mdayican) [reply docker-compose-dependencies.yml:19]: normally yes, it is a fix.

**Pull request merged**